app/views/govuk_component/metadata.raw.html.erb

```diff
@@ -16,13 +16,23 @@
       <dt><%= t("govuk_component.metadata.from", default: "From") %>:</dt>
       <dd><%= from.to_sentence.html_safe %></dd>
     <% end %>
+    <% if part_of.any? %>
+      <dt>Part of:</dt>
+      <dd><%= part_of.to_sentence.html_safe %></dd>
+    <% end %>
     <% if local_assigns.include?(:history) %>
       <dt>History:</dt>
       <dd><%= history %></dd>
     <% end %>
-    <% if part_of.any? %>
-      <dt>Part of:</dt>
-      <dd><%= part_of.to_sentence.html_safe %></dd>
+    <% if local_assigns.include?(:first_published) && first_published %>
+      <dt>First published:</dt>
+      <dd><%= first_published %></dd>
+    <% end %>
+    <% if local_assigns.include?(:last_updated) && last_updated %>
+      <dt>Last updated:</dt>
+      <dd>
+        <%= last_updated %><% if local_assigns.include?(:see_updates_link) %>, <a href="#history">see all updates</a><% end %>
```

Comment:
My worry is the dependency on another component being on the page, by making it an explicit thing to include this doesn't accidentally lead to broken links.